yt_dlp/__init__.py

```diff
@@ -1,8 +1,8 @@
-try:
-    import contextvars  # noqa: F401
-except Exception:
-    raise Exception(
-        f'You are using an unsupported version of Python. Only Python versions 3.7 and above are supported by yt-dlp')  # noqa: F541
+import sys
+
+if sys.version_info < (3, 8):
```

Comment:
https://github.com/yt-dlp/yt-dlp/pull/8361/files#r1388945992

```suggestion
try:
    import contextvars  # noqa: F401
except Exception:
```